pandas/tests/test_frame.py

```diff
@@ -13591,6 +13592,67 @@ def test_quantile_axis_parameter(self):
         self.assertRaises(ValueError, df.quantile, 0.1, axis=-1)
         self.assertRaises(ValueError, df.quantile, 0.1, axis="column")
 
+    def test_quantile_interpolation(self):
+        # GH #10174
+        from numpy import percentile
+
+        #interpolation = linear (default case)
+        q = self.tsframe.quantile(0.1, axis=0,interpolation='linear')
+        self.assertEqual(q['A'], percentile(self.tsframe['A'], 10))
+        q = self.intframe.quantile(0.1)
+        self.assertEqual(q['A'], percentile(self.intframe['A'], 10))
+                
+        q1 = self.intframe.quantile(0.1)
+        self.assertEqual(q1['A'], np.percentile(self.intframe['A'], 10))        
+        #test with and without interpolation keyword
+        assert_series_equal(q,q1)   
+        
+        #interpolation method other than default linear
```

Comment:
make this 2 tests, 1 for the version checking, and the other that would skip at the top if under verion 1.9